pywbem_mock/_baseprovider.py

```diff
@@ -88,6 +76,16 @@ def __init__(self, cimrepository=None, provider_registry=None):
             repository is fully initialized.
 
           provider_registry (dict): Provider registry.
+
```

Comment:
This parameter list should describe the input parameters for `__init__()`, but `provider_type` and `provider_class` are no such input parameters. The static attributes with these names are currently described in _defaultinstanceprovider.py , so I suggest to consolidate the description there.

Reply:
DONE